arviz/plots/backends/bokeh/distplot.py

```diff
@@ -105,6 +115,10 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
         hist_kwargs["fill_color"] = color
         hist_kwargs["line_color"] = color
 
+    line_alpha = hist_kwargs.pop("line_alpha", False)
+    if not line_alpha:
+        hist_kwargs.setdefault("line_alpha", 0)
```

Comment:
This will remove `"line_alpha"` and if not found, set it to 0?

Would this work?

```suggestion
    hist_kwargs.setdefault("line_alpha", 0)
```